CHANGELOG.md

```diff
@@ -15,7 +15,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
```

Comment:
Shouldn't there an entry saying that it was removed? I'd be in favor of keeping the history of the log.

Reply:
You are completely right! Done :)